samples/nrf9160/modem_shell/overlay-location_service_ext.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Modem Shell
+CONFIG_MOSH_CLOUD_MQTT=y
+CONFIG_MOSH_CLOUD_REST=n
```

Comment:
I would use already existing mqtt overlay for these two configs as the intention here is to enable MQTT or at least rename the file to include cloud-mqtt. 
I think that it would be also ok to have it like (eventhough there would be just CONFIG_LOCATION_SERVICE_EXTERNAL in overlay):

`west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG="overlay-cloud_mqtt.conf;overlay-location_service_ext.conf`

Reply:
We'll resolve this in comment https://github.com/nrfconnect/sdk-nrf/pull/9804#discussion_r1066233239.

I'm leaning towards removing this overlay altogether and instructing to use cloud_mqtt overlay and separately specifying `CONFIG_LOCATION_SERVICE_EXTERNAL=y`:
`west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG="overlay-cloud_mqtt.conf" -DCONFIG_LOCATION_SERVICE_EXTERNAL=y`